src/components/AccessibilityPolicyForm/index.jsx

```diff
@@ -0,0 +1,278 @@
+      if (!inputValue) {
+        feedback = {
+          isValid: false,
+          validationMessage: 'Full name required.',
+        };
+      }
+    } else if (inputType === 'message') {
+      if (!inputValue) {
+        feedback = {
+          isValid: false,
+          validationMessage: 'Message required.',
+        };
+      }
+    }
+
+    return feedback;
+  }
+
+  closeStatusAlert() {
+    this.emailInputRef.focus();
+    this.setState({
+      statusAlertOpen: false,
+    });
+  }
+
+  handleChange(value, name) {
+    if (name === 'email') {
+      this.setState({
+        formEmail: value,
+      });
+    } else if (name === 'fullName') {
+      this.setState({
+        formFullName: value,
+      });
+    } else if (name === 'message') {
+      this.setState({
+        formMessage: value,
+      });
+    }
+  }
+
+  renderStatusAlert() {
+    const status = this.getStatusFields();
+
+    const statusAlert = (
+      <StatusAlert
+        alertType={status.alertType}
+        dialog={status.alertDialog}
+        open={this.state.statusAlertOpen}
+        onClose={this.closeStatusAlert}
+        ref={(input) => { this.statusAlertRef = input; }}
+      />
+    );
+
+    return (
+      <div>
+        {statusAlert}
+      </div>
+    );
+  }
+
+  render() {
+    return (
+      <div>
+        <h1 id="a11y-feedback" className={styles['page-header']}>Studio Accessibility Feedback</h1>
+        {this.renderStatusAlert()}
+        <h6>All fileds required.</h6>
+        <section className={styles['form-section']} role="group" aria-labelledby="a11y-feedback">
+          <InputText
+            name="email"
+            label="Your Email"
+            id="email"
+            type="email"
+            onChange={this.handleChange}
+            value={this.state.formEmail}
+            themes={['danger']}
+            validator={email => this.validateFormContent('email', email)}
+            inputRef={(ref) => { this.emailInputRef = ref; }}
+          />
+          <InputText
+            name="fullName"
+            label="Full Name"
+            id="fullName"
+            onChange={this.handleChange}
+            value={this.state.formFullName}
+            themes={['danger']}
+            validator={fullName => this.validateFormContent('fullName', fullName)}
+            inputRef={(ref) => { this.fullNameInputRef = ref; }}
+          />
+          <TextArea
+            name="message"
+            label="Message"
+            id="message"
+            onChange={this.handleChange}
+            value={this.state.messageInputRef}
+            themes={['danger']}
+            validator={message => this.validateFormContent('message', message)}
+            inputRef={(ref) => { this.messageInputRef = ref; }}
+          />
+          <Button
+            buttonType="primary"
+            label="Submit"
+            aria-label="Submit Accessibility Feedback Form"
+            onClick={() => { this.onSubmitClick(); }}
```

Comment:
So correct me if I'm wrong but it looks each of the form areas have their own validators, and then we also do the same validation in `onSubmitClick`, right? Why not block the ability to click this button at all if any of the text areas have invalid inputs?

Reply:
There are different validation view updates -- the input box changes onBlur to validate (border and validation message below) - but there was also an ask for a view of a Status Alert that will also call out any errors when trying to submit that will alert the users that things are incorrect and to go back and resolve.